pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt.go

```diff
@@ -153,14 +177,21 @@ func (handler DeviceCommandHandlerMQTT) commandHandleFunc() http.HandlerFunc {
 				return
 			}
 		} else if reqType == http.MethodPost {
+			mqttTopic := handler.HandlerMetaData.properties.MQTTTopic
+			logger.Infof("the mutexInstructions is %v", mutexInstructions)
+			if mutexBlockingMap[mqttTopic] {
+				blockedMessage := "MQTT broker is blocked"
```

Comment:
shall we put the topic and the time in the message? 
eg. fmt.Printf("MQTT topic %s is blocked at %s",mqttTopic, time.Now())